modules/services/window-managers/i3-sway/sway.nix

```
@@ -139,6 +139,15 @@ let
         '';
       };
 
+      bindkeysToCode = mkOption {
+        type = types.bool;
+        default = false;
+        example = true;
+        description = ''
+          Whether to make use of `--to-code` in keybindings";
```

Need to use docbook markup. Also there is a stray `";`.
```suggestion
          Whether to make use of <option>--to-code</option> in keybindings.
```